.env.example

```
@@ -0,0 +1,142 @@
+
+# Text Chunking Configuration
+CHUNK_SIZE=800
+CHUNK_OVERLAP=100
+MAX_CONTEXT_TOKENS=2000
+
+# =============================================================================
+
+# WEB CRAWLER & ENRICHMENT
+# =============================================================================
+
+CRAWLER_USER_AGENT=user-doc-chat/1.0 (+enrichment)
```

dotenv-linter (3.3.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)